baremetal/metal3machine_manager.go

```diff
@@ -535,8 +545,26 @@ func (m *MachineManager) Delete(ctx context.Context) error {
 			host.Spec.NetworkData = nil
 			bmhUpdated = true
 		}
-		if host.Spec.Online {
+
+		//	Change bmh's online status to on/off  based on AutomatedCleaningMode and Capm3FastTrack values
+		//	AutomatedCleaningMode |	Capm3FastTrack|   BMH
+		//		disabled				false 			turn off
+		//		disabled				true 			turn off
+		//		metadata				false 			turn off
+		//		metadata				true 			turn on
```

Comment:
Not critical, but aligning the fields could improve readability.

Reply:
I tried to align lines for both code editor and for github, this was max I came up with. If I make alignment for one, in the other it will be unaligned.